README.md

```diff
@@ -247,6 +246,9 @@ options:
                         Set for dry run (default: none)
   -o {yaml,none}, --output {yaml,none}
                         Set to get output of resources which are created (default: none).
+  -j SERVICE_ACCOUNT_NAMESPACE, --service_account_namespace SERVICE_ACCOUNT_NAMESPACE
```

Comment:
should this be `service-account-namespace`?
```suggestion
  -j SERVICE_ACCOUNT_NAMESPACE, --service-account-namespace SERVICE_ACCOUNT_NAMESPACE
```

Reply:
yes good catch on this, updated it